src/helper_generic.py

```diff
@@ -1,21 +1,22 @@
 """
-Helper Generic perform generic oprations for threading.
+Helper Generic perform generic operations for threading.
 
 Also perform some conversion operations.
 """
 
+
 import socket
 import sys
-from binascii import hexlify, unhexlify
```

Comment:
Here also place imports in a order.

Reply:
multiprocessing is special, importing it in wrong order could cause some initialisation to be executed in wrong order. The initialisation has to be fixed first.